spec/mailers/csv_submitted_for_processing_first_stage_email_mailer_spec.rb

```diff
@@ -16,12 +16,12 @@
     let(:template_ids) do
       {
         in_progress: "in-progress-uuid",
-        succeeded: "succeeded-uuid",
+        validated: "validated-uuid",
         failed: "failed-uuid",
       }
     end
 
-    %i[in_progress succeeded failed].each do |status|
+    %i[in_progress validated failed].each do |status|
```

Comment:
Should the `pending` status be here too?
```suggestion
    %i[pending validated failed].each do |status|
```